### PR TITLE
Changed import menu to specify real and expected timing files

### PR DESCRIPTION
**Pull request by TijnBertens**
#### What have I done
Changed the import menu to indicate what imports are for the real and expected timing files. This is still a part of fixing issue #434.

#### Testing
Automated testing.
Check that the import menu shows a real and expected import and when either or both are selected, it is possible to import.

#### Addresses SRD section(s)
Import

### CODE REVIEW

File: src/app/import/import.component.ts [hunk 1]
```diff
@@ -172,16 +166,16 @@ export class ImportComponent implements OnInit {
...
+        this.timingFileReal = files;
     }
 
     /**
      * Sets the second timing file.
      * @param files array containing the one file that needs to be set
      */
-    inputTimingTwo(files: FileList) {
-        this.timingFileTwo = files;
+    inputTimingExpected(files: FileList) {
```
Comment (Erik9519) [inline src/app/import/import.component.ts]: Missing return type

File: src/app/import/import.component.ts [hunk 2]
```diff
@@ -172,16 +166,16 @@ export class ImportComponent implements OnInit {
      * Sets the first timing file.
      * @param files array containing the one file that needs to be set
      */
-    inputTimingOne(files: FileList) {
-        this.timingFileOne = files;
+    inputTimingReal(files: FileList) {
```
Comment (Erik9519) [inline src/app/import/import.component.ts]: Missing return type

File: src/app/import/import.component.ts [hunk 3]
```diff
@@ -237,18 +231,17 @@ export class ImportComponent implements OnInit {
      */
     clearFiles() {
```
Comment (Erik9519) [inline src/app/import/import.component.ts]: Missing return type

File: src/app/import/import.component.ts (hunk 1 above)
Comment (TijnBertens) [reply src/app/import/import.component.ts]: Fixed

File: src/app/import/import.component.ts (hunk 3 above)
Comment (TijnBertens) [reply src/app/import/import.component.ts]: Fixed

File: src/app/import/import.component.ts (hunk 2 above)
Comment (TijnBertens) [reply src/app/import/import.component.ts]: Fixed

**Pull request merged**